doc/users/whats_new/imshow_logscales.rst

```diff
@@ -0,0 +1,30 @@
+Log-scale on image plots
+------------------------
+
+:func:`imshow` not properly displays data on log-scales.
```

Comment:
You're right on both accounts (the `if 1:` trick is there so can I can lazy when copy/pasting into jupyter -- oops)